functions/transformations/max.go

```diff
@@ -131,33 +131,39 @@ func (s *MaxSelector) selectRow(idx int, cr flux.ArrowColReader) {
 
 func (s *MaxIntSelector) DoInt(vs *array.Int64, cr flux.ArrowColReader) {
 	maxIdx := -1
-	for i, v := range vs.Int64Values() {
-		if !s.set || v > s.max {
-			s.set = true
-			s.max = v
-			maxIdx = i
+	for i := 0; i < vs.Len(); i++ {
```

Comment:
I guess you could check to see if there are any nulls first and if there are none, have a tighter loop for that case.  But this is probably fine.

Reply:
Related to this comment in the min version of this PR: https://github.com/influxdata/flux/pull/690#discussion_r246133915